src/core/__tests__/QueryManager/index.ts

```diff
@@ -5228,5 +5228,166 @@ describe('QueryManager', () => {
 
       expect(queryManager['inFlightLinkObservables'].size).toBe(0)
     });
-  })
+  });
+
+  describe('warnings', () => {
```

Comment:
```suggestion
  describe('missing cache field warnings', () => {
```
A little more specific?